sources/dylan/thread.dylan

```diff
@@ -30,6 +30,8 @@ define sealed class <thread> (<portable-double-container>)
 
 end class;
 
+ignore(priority);
```

Comment:
@vendethiel 

It does ignore the argument, CL-style. It works because a slot declaration creates a generic function (well, a mutable slot generates 2: ``priority`` and ``priority-setter``, a ``constant slot`` just generates the first).

Since nothing on the Dylan side is using that ``priority`` slot, we get a warning.

I thought about renaming the slot to ``thread-priority`` and then exporting it, but that seemed like a good step for another time if it is needed.